Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/ShapeTranslationOffsetViewportEdit.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Component/ComponentBus.h>
+#include <AzToolsFramework/ComponentModes/BaseViewportEdit.h>
+#include <AzToolsFramework/Manipulators/TranslationManipulators.h>
+
+namespace AzToolsFramework
+{
+    //! Wraps translation manipulators for editing shape translation offsets.
+    class ShapeTranslationOffsetViewportEdit : public BaseViewportEdit
```

Comment:
This interface and implementation is really nice, awesome to be able to bundle up this shared logic 👍